masonry/src/properties/box_shadow.rs

```diff
@@ -78,7 +80,7 @@ impl BoxShadow {
         if property_type != TypeId::of::<Self>() {
             return;
         }
-        // TODO - request_paint_only?
+        // TODO - Call request_post_paint instead, which should be lighter?
         ctx.request_layout();
```

Comment:
Given how localised this change would be, it's not clear why it's not been done in this PR.

Perhaps this should be replaced with "TODO: determine whether `request_post_paint` would be correct here, as it would be more performant".